### PR TITLE
Updated for Alpha 2 release.

### PR DESCRIPTION
**Pull request by jclingan**

### CODE REVIEW

File: src/app/wizard/versions/versions.md [hunk 1]
```diff
@@ -2,8 +2,33 @@
...
 ## Overview
 
-Obsidian Toaster is a cloud application platform that includes a collection of cloud services with language runtime integration running on OpenShift.
-This initial release delivers a project generator that creates a quickstart or custom Spring Boot / Vert.x starter project, optionally secured with Red hat SSO, and is ready to be built and deployed to OpenShift.
+Obsidian Toaster defines and implements an opinionated approach to creating and running polyglot applications optimized for a common set of services running on OpenShift.
+
+## What's New in Alpha 2
+
+__WildFly Swarm__
+Adds WildFly Swarm (version 2017.1.1) as a language runtime.
```
Comment (gastaldi) [inline src/app/wizard/versions/versions.md]: The version used in the Starter project is 2017.2.0, maybe the quickstarts should follow the same version?

**Comment by jclingan** [comment]
What is the best way of removing references to the WildFly Swarm JDBC quickstart from the release notes? The pull request we approved but not merged. Should I wait for this to be merged?

**Comment by gastaldi** [comment]
You can edit and commit in your branch again, then the PR is updated

**Comment by gastaldi** [comment]
https://github.com/jclingan/generator-frontend/blob/patch-1/src/app/wizard/versions/versions.md

File: src/app/wizard/versions/versions.md (hunk 1 above)
Comment (gastaldi) [inline src/app/wizard/versions/versions.md]: I've updated the Quickstarts to run on 2017.2.0

File: src/app/wizard/versions/versions.md [hunk 2]
```diff
@@ -12,33 +37,59 @@ This initial release delivers a project generator that creates a quickstart or c
...
 
+### Eclipse Vert.x - ConfigMap Quickstart
+Simple REST endpoint where the Vert.x container uses kubernetes ConfigMap to get the Application parameters from a configuration file mounted.
+
+### Eclipse Vert.x Starter
 Generate a custom Vert.x project template by selecting from a list of available Vert.x modules.
 The project template includes a simple HTTP Server that can be easily deployed to OpenShift using a maven plugin.
 
+### WildFly Swarm - REST Quickstart
+Expose a REST Greeting endpoint using Eclipse Vert.x
```
Comment (gastaldi) [inline src/app/wizard/versions/versions.md]: Is this description correct?

**Comment by jclingan** [comment]
Ahhh. Then you have to approve again?

**Comment by gastaldi** [comment]
Yes, I'll review it and if it looks OK I can merge

**Comment by gastaldi** [comment]
Great job! Thanks!

**Pull request merged**